crates/solidity/testing/sanctuary/src/main.rs

```diff
@@ -92,7 +92,8 @@ fn main() -> Result<()> {
     if failure_count > 0 {
         println!();
         println!(
-            "Found '{failure_count}' failures. Please check the logs above for more information."
+            "Found {failure_count} failure{suffix}. Please check the logs above for more information.",
```

Comment:
nit suggestion: using the existing `Inflector` `to_plural()` helper/

Reply:
I don't think

```rust
println!(
            "Found {failure_count} {failures}. Please check the logs above for more information.",
            failures = if failure_count == 1 { "failure" } else { &"failure".to_plural() },
        );
```

reads better. Does it? Do you want me to use this, instead?

Reply:
Apologies. I was thinking of NPM's `pluralize` which automatically handles that: https://www.npmjs.com/package/pluralize

Please feel free to ignore this then, or simply do `"Found {failure_count} failure(s)."` if you like :)